app/assets/javascripts/external_activities/matedit.js

```diff
@@ -1,27 +1,31 @@
 function get_edit_resource_popup(external_activity_id, popup_options){
   var lightboxConfig = {
         content:"<div style='padding:10px'>Loading...Please Wait.</div>",
-        title:"Edit External Activity"
+        title:"Edit Options"
     };
     list_lightbox=new Lightbox(lightboxConfig);
-    jQuery(".n.move_handle").css({"width":"400px"});
     jQuery(".buttons")[0].innerHTML = jQuery(".buttons")[0].innerHTML + "<a href='javascript:void(0)' style='float:right;margin-top: 5px;margin-right: 10px' class='button' onclick='close_popup()'>Cancel</a>"
     var target_url = "/eresources/999/edit";
     if (popup_options && popup_options.use_short_form) {
       target_url = "/eresources/999/edit_basic"
     }
     target_url = target_url.replace('999',external_activity_id);
```

Comment:
Ah yes. I remember now.  I'll add a comment.
And I need to try to remember that because of the pre-compile issue we can use ruby code in our assets.

So in this case the better approach would be to add these URLs to a JSON description of the external activity and pass them into this method.